canonical-websites/www.ubuntu.com
# Add server-provisioning to /server/maas redirect

## Done

Add server-provisioning to server/maas redirect

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/server-provisioning](http://0.0.0.0:8001/server-provisioning)
- Make sure you're redirected to /server/maas

## Issue / Card

Fixes #2431
